features/graph/documentation/get-started.md

```diff
@@ -0,0 +1,98 @@
+  You can set **Authentication** scopes by selecting the **Authentication** dropdown menu.
+5. Under **Open sessions**, select a session to start building GraphQL queries for your connected API or service.
+
+## Start a local CLI session with Netlify Dev
+
+To make full use of Netlify Graph, you need Netlify Dev and the Netlify CLI. Make sure you install Netlify CLI and have the latest version.
+
+``` sh
+npm install netlify-cli -g
+```
+
+To start a new Graph session:
+1. In your terminal, navigate to your site's local directory.
+2. Run `netlify dev --graph` to start a local development server.
+
+To learn more about the `graph` CLI commands, refer to our [Netlify CLI documentation](https://cli.netlify.com/commands/graph/).
+
+## Create a GraphQL query with Graph Explorer
+
+The Graph Explorer is a visual editor that helps you write GraphQL queries, mutations, subscriptions, and fragments.
+
+To create a new query:
+1. Make sure you have an [open session](#start-a-local-cli-session-with-netlify-dev).
+2. In the Netlify UI, select your site, then navigate to the **Graph** tab. 
+3. Under **Open sessions**, select your current session.
+4. On the Graph Explorer page, select **Create new**, then **Query**.
+5. Write a description for your query.
+6. Use the **Explorer** to name your query and start composing.
+7. When you're done composing, select **Save changes**.
+
+## Generate a query handler
+With Graph Explorer, you can auto-generate handlers for your queries. These handlers are wrapped in Netlify Functions that you can call in your project. 
+
+To generate a query handler:
+1. In the Netlify UI, select your site, then navigate to the **Graph** tab.
+2. [Start a new session with Netlify Dev](#start-a-new-session-with-netlify-dev) or select a session under **Open sessions**
+3. On your session page, select **Query library** and find the query you want to generate handlers for.
+4. On the right-hand side of the page, select **Options**, then **Generate handler**.
+5. Update your local copy by running `netlify graph:pull`.
+
+Auto-generated handlers are isolated to your CLI session until you commit them to your repo.
+
+## Use a query handler in your project
+
+After generating a query handler and pulling it to your local project, you can access that handler in your project's code. Handlers are called just like other [Netlify Functions](https://docs.netlify.com/functions/build-with-javascript/).
+
+Queries are stored in your local project under `netlify/functions/YOUR_QUERY`, which means that you need to call their endpoints like this: `/.netlify/functions/YOUR_QUERY`. Note the preceding slash and period.
+
+You can test your query handlers locally by running `netlify dev --graph` at your project's root.
+
+To deploy the functions, commit the changes to your local project and push to your git remote origin. This starts a deploy of your Netlify site and its associated functions.
+
+<!-- QUESTION: Is this still relevant? -->
```

Comment:
@localden, the 2 following paragraphs don't seem relevant anymore. But maybe I'm missing something. Do users still need to pass an auth token with their calls to the generated handlers?

Reply:
@KyleBlankRollins no, they are still relevant. We don't auto-bind authentication to generated code.